### PR TITLE
Enable the language selector for students and teachers.

### LINKED ISSUE #3239: As student, no way to set my own default language
branch: develop

As a student, we used to have the option to go to the "Edit User" page, which included the option to edit our own default language. 

screen shot 2015-03-10 at 2 17 56 pm

However, since we disabled that, students now have no way to set their own language. No language dropdown in sight too.

### PR DESCRIPTION
**Pull request by cpauya**
Hi @aronasorman - this fixes #3239 to Enable the language selector for students and teachers.

See .gif how it works:

ka-lite

### CODE REVIEW

**Comment by aronasorman** [comment]
There's 1 test failure, ironically about the dropdown not showing up for coaches!

**Comment by MCGallaspy** [comment]
Probably broke the selector for that test.

File: kalite/distributed/templates/distributed/base.html [hunk 1]
```diff
@@ -302,6 +291,22 @@
                                 </li>
                               </ul>
                             </li>
+                            <!-- Language selection -->
+                            {% if request.session.facility_user.is_teacher %}
+                            <li class="teacher-only">
+                            {% else %}
+                            <li class="not-admin-only">
+                            {% endif %}
```
Comment (MCGallaspy) [inline kalite/distributed/templates/distributed/base.html]: Shouldn't this be visible for everyone? I think you can replace this conditional with just a `<li>` tag (no class).

File: kalite/distributed/templates/distributed/base.html [hunk 2]
```diff
@@ -302,6 +291,22 @@
...
+                              {% if num_language_choices > 1 %}
+                              <select id="language_selector">
+                              {% for lang_code, lang_meta in language_choices.iteritems %}
+                                <option value="{{ lang_code }}" {% if lang_code == current_language %}selected{% endif %}>
+                                    {{ lang_code|get_language_name }}
+                                </option>
+                              {% endfor %}
+                              </select>
+                              {% endif %}
```
Comment (MCGallaspy) [inline kalite/distributed/templates/distributed/base.html]: Can you indent the if block, for readability?

**Comment by MCGallaspy** [comment]
Yeah, regarding the failed test, it selects the dropdown by xpath. @cpauya can you just update the xpath selector in that test?

File: kalite/distributed/templates/distributed/base.html (hunk 1 above)
Comment (cpauya) [inline kalite/distributed/templates/distributed/base.html]: No @MCGallaspy - as discussed with @jamalex and @aronasorman - the language dropdown will not be shown for the admins.

File: kalite/distributed/templates/distributed/base.html (hunk 2 above)
Comment (cpauya) [inline kalite/distributed/templates/distributed/base.html]: Yikes, thanks for noticing, in fact I might just as well do the `if` on the class.

**Comment by cpauya** [comment]
Hi @aronasorman / @MCGallaspy - I've fixed the tests and template changes.  Please check.

**Comment by aronasorman** [comment]
Can confirm that the dropdown is now there.

**Pull request merged**